src/transforms/deBarrelify/replaceImportDeclarationWithDeepImport/replaceImportDeclarationWithDeepImport.js

```diff
@@ -103,7 +109,7 @@ const replaceImportDeclarationWithDeepImport = ({
     builder.importDeclaration(
       [newSpecifier],
       builder.stringLiteral(finalizedImportSource),
-      path.node.importKind
+      specifier?.importKind
```

Comment:
It seems `path.node.importKind` gets `value` even when it's `type`.

Reply:
For example, `./src/store/catalogFilters/useUpdateCatalogFilters/useUpdateCatalogFilters.ts` 

```
import { useSetCatalogFilters, type Filters, type Filter } from "@store/catalogFilters";
```

`path.node.importKind` returns `value`,  `specifier?.importKind` returns `type`. 

the current js-transform generates this

```
import { Filter } from "@store/catalogFilters/catalogFilters";
import { Filters } from "@store/catalogFilters/catalogFilters";
import { useSetCatalogFilters } from "@store/catalogFilters/catalogFilters";
```

but prettier will add `type`.

Also there are opposite cases too.  
`path.node.importKind` returns `type`,  `specifier?.importKind` returns `value`.

So probably this fix isn't right. will revert this fix for now